src/Illuminate/Queue/Console/ListenCommand.php

```diff
@@ -63,6 +63,12 @@ public function fire()
 
         $timeout = $this->input->getOption('timeout');
 
+        $sleep = $this->input->getOption('sleep');
+
+        if ($timeout <= $sleep) {
+            throw new \InvalidArgumentException("--timeout value must be higher than --sleep");
```

Comment:
`throw new InvalidArgumentException('The timeout must be greater than the sleep time.');`